lib/app/services/network.dart

```diff
@@ -70,122 +68,108 @@ class TooManyRequestsError extends ServerError {
   final Duration timeToWait;
 }
 
-/// A service that offers networking HTTP requests to the backend servers.
-///
-/// It depends on the authentication storage, so if the user's token
-/// is stored there, the requests' headers are automatically enriched with the
-/// access token.
+/// A service that offers networking POST and GET requests to arbitrary
```

Comment:
Only POST and GET requests is no longer correct—maybe just say "network requests"?